src/helpers/buildWalletSections.js

```diff
@@ -165,7 +165,7 @@ const withBriefUniswapSection = (
 
 const withBalanceSavingsSection = (savings, network) => {
```

Comment:
and no need for the `?.map`
```suggestion
const withBalanceSavingsSection = (savings = [], network) => {
```